UPGRADE.md

```diff
@@ -223,6 +223,7 @@ class BookController extends ResourceController
 ### Core and CoreBundle
 
 * Moved ``taxCategory`` from ``Product`` to ``ProductVariant``
+* Set ``OrderItem`` not to be ``PromotionSubject``
```

Comment:
Set not to be... :dancer: 

* ``OrderItem`` model from Core component no longer implements ``PromotionSubjectInterface``, remove all usages of ``addPromotion``, etc. from your custom code. Custom actions targeting items need to be adjusted, see ``ContainsProductRuleChecker`` for reference.